chia/consensus/multiprocess_validation.py

```diff
@@ -102,11 +112,29 @@ def batch_pre_validate_blocks(
                 if error is not None:
                     error_int = uint16(error.code.value)
```

Comment:
if we get here, and set `error_int`, shouldn't we return?
as it is right now, it looks like we might overwrite this (true) error code with `Err.BAD_AGGREGATE_SIGNATURE` later

Reply:
it would be interesting to test this function with an invalid block but with a valid signature

Reply:
Yes good point, the signature error will take precedence, kind of like if the signature was validated before the CLVM. It shouldn't cause the block to be valid though. If the `PreValidationResult` has an error, the block is completely invalid and will be thrown out. 

Still it does seem better to just add the error and continue instead of force validating which could be slow